apps/api/src/auth/supertokens/sms.service.ts

```diff
@@ -0,0 +1,38 @@
+// @ts-ignore
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { catchError, firstValueFrom } from 'rxjs';
+
+@Injectable()
+export class SmsService {
+  constructor(private readonly httpService: HttpService) {}
+
+  async smsSend(phone: string, otp: string) {
+    const headerValues = {
+      authkey: process.env.MSG91_AUTH_KEY as string,
+      'content-type': 'application/json' as string,
+    };
+
+    await firstValueFrom(
+      this.httpService
+        .post(
+          process.env.MSG91_API_DOMAIN as string,
+          {
+            flow_id: process.env.MSG91_FLOW_ID,
```

Comment:
Dont use process.env. 

Use the [config](https://docs.nestjs.com/techniques/configuration#using-the-configservice) service to access env variables

Reply:
Alternatively you can set config module [globally](https://docs.nestjs.com/techniques/configuration#using-the-configservice) instead of importing the module in each other